content/docs/user-guide/managing-external-data.md

```diff
@@ -15,14 +15,13 @@ versioning existing data on a network attached storage (NAS), processing data on
 HDFS, running [Dask](https://dask.org/) via SSH, or any code that generates
 massive files directly to the cloud.
 
-External outputs (and
+_External outputs_ (and
 [external dependencies](/doc/user-guide/external-dependencies)) provide ways to
 track and version data outside of the <abbr>project</abbr>.
 
 ## How external outputs work
 
-External <abbr>outputs</abbr> are considered part of the (extended)
-<abbr>workspace</abbr>: DVC will track them for
+External <abbr>outputs</abbr> will be tracked by DVC for
```

Comment:
does the `<abbr>` (https://github.com/iterative/dvc.org/blob/master/content/docs/user-guide/basic-concepts/dvc-cache.md) need updating?

Reply:
I don't think so. The tooltip says "_by default_" and it implies it can be configured to have different locations, e.g., externally in some NFS mount.

Reply:
But the page needs a concept explanation.

Reply:
Did you mean the `output` tooltip @casperdcl ? (I edited your comment). If so can you specify why or what part? Sorry, the connection is not obvious to me 🙂 The part about .dvc files?

> But the page needs a concept explanation.

@iesahin you mean a standalone concepts page? Sure, but out of scope here 😬 (related to #2453 though)

Reply:
oops, yes wrong link. I'm complaining about a different `<abbr>` (output) here.

The point is the title was changed from "external outputs" to "external data" and yet we continue using "external outputs" here, while the tooltip is only about "outputs."

Reply:
OK.

1. Context on why we don't want the page title to have the word "output": https://github.com/iterative/dvc.org/pull/2154#pullrequestreview-610353476
2. True. But the section name didn't change and it is **How external outputs work**.
3. Right, so similar to the "external cache" comment where you see possible confusion by adding an adjective to the tooltipped term? TBH I don't see it but maybe I'm biased.

Reply:
I have the approach where docs are not meant for *us*, they're meant for people who are confused are thus looking for docs. I understand, but I don't think they will. Using multiple different similar-sounding terms tends to create a sense of complexity and makes readers (consciously or subconsciously) feel lost.

Reply:
Yeah of course they're not meant for us 🙂 

I just don't understand how this can be confusing. What are the similar sounding terms here? OR if you have a suggestion to improve this, maybe that would clarify your PoV.